cfn-resources/cloud-backup-restore-jobs/cmd/resource/resource.go

```diff
@@ -482,10 +487,11 @@ func getRestoreJob(client *mongodbatlas.Client, currentModel *Model) (*mongodbat
 }
 
 func isJobFinished(job mongodbatlas.CloudProviderSnapshotRestoreJob) bool {
-	failed := job.Failed != nil && *job.Failed
-	finished := job.FinishedAt != ""
+	return job.FinishedAt != ""
+}
 
-	return failed || job.Cancelled || job.Expired || finished
+func isJobFailed(job mongodbatlas.CloudProviderSnapshotRestoreJob) bool {
+	return job.Failed != nil && *job.Failed
```

Comment:
why don't we have Failed in the model?  I think it would be easier, e.g. we wouldn't need isJobFailed to use the API object and could just use the model

Reply:
apart of adding this property, we have to think if is going to be a valid property for the user, if is going to be used only here i dont see a reason to add it to the model